tests/unit/utils/fix-special-char-test.js

```diff
@@ -0,0 +1,10 @@
+import fixSpecialChar from 'dummy/utils/fix-special-char';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | fix special char');
+
+// Replace this with your real tests.
```

Comment:
As a general nudge- it's a good idea to remove auto-generated placeholder text like this before you commit a branch.